full_rap_apply.py

```diff
@@ -0,0 +1,189 @@
+import yaml, glob
```

Comment:
Can you remove this file since lit looks like a duplicate of the one in the ptype folder?